template/style.css

```
@@ -0,0 +1,205 @@
+  justify-content:center;
+}
+
+.btn{
+  appearance:none;
+  border:0;
+  padding: .7rem 1rem;
+  border-radius: 12px;
+  font-weight: 600;
+  cursor:pointer;
+  transition: transform .05s ease, filter .15s ease, background-color .15s ease, opacity .15s ease;
+  border: 1px solid rgba(255,255,255,.08);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)